nototools/generate_website_2_data.py

```diff
@@ -59,6 +58,13 @@
 APACHE_LICENSE_LOC = path.join(FONTS_DIR, 'LICENSE')
```

Comment:
Is FONTS_DIR the top of noto-fonts dir? If so,  LICENSE file there is also OFL, isn't it? 
LICENSE file in noto-emoji is Apache, but for all other fonts, LICENSE is OFL 1.1.